lib/core/alerts.ts

```diff
@@ -1,9 +1,54 @@
 import chalk from "chalk";
 
-const error = (message: string) => console.log(chalk.red(message));
-const warn = (message: string) => console.log(chalk.yellowBright(message));
-const notice = (message: string) => console.log(chalk.gray(message));
-const info = (message: string) => console.log(chalk.blueBright(message));
-const success = (message: string) => console.log(chalk.green(message));
+export type LogLevel = "verbose" | "error" | "info" | "silent";
+export const LOG_LEVELS: LogLevel[] = ["verbose", "error", "info", "silent"];
```

Comment:
If you wanted, we could de-dupe this a little bit with type inference:

```typescript
export const LOG_LEVELS = ["verbose", "error", "info", "silent"] as const;
export type LogLevel = typeof LOG_LEVELS[number]
```

Now we only need to change `LOG_LEVELS` and the type will reflect that.